app/src/main/res/layout/activity_main.xml

```diff
@@ -44,4 +50,12 @@
         android:layout_height="wrap_content"
         android:text="@string/buttonViewLayersText" />
 
+        <Button
```

Comment:
No perdy trash can? lol

Reply:
Button was in use for testing and not actually the method used to delete layers.
Removed.